source/wind_energy.rst

```diff
@@ -1,8 +1,8 @@
 .. _wind-energy:
 
-*******************************
-Offshore Wind Energy Production
-*******************************
+**********************
+Wind Energy Production
+**********************
```

Comment:
I agree and think that we should cast a wider net of reviewers for some of these changes. I mentioned it in a comment before but there's a lot of history with the UG and how things ended up being labeled. I'd want to make sure we got at least Lisa in here to review, if not others.

@rmgriffin

Reply:
As set up it is definitely an offshore wind model. I'm pretty sure it could be coerced to do terrestrial wind modeling, though I don't think it would be a satisfying experience. Are there other changes to this model, or others like wave energy or valuation text generally that need review?

Reply:
Thanks @rmgriffin, I don't think this PR is touching any substance other than the Theme and trying to have the UG reflect what people find in the UI. We will certainly tag you for wind, wave, and valuation in general if it comes up in the future!